@AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/weapon/fn_weaponTurretAim.sqf

```diff
@@ -27,7 +27,7 @@ private _inhibit = "";
 private _weaponsProcessorFailed = _heli getVariable "fza_ah64_rwp_fail" && _heli getVariable "fza_ah64_lwp_fail";
 
 private _usingRocket = currentweapon _heli isKindOf["fza_hydra70", configFile >> "CfgWeapons"] || currentWeapon _heli == "fza_rkt_safe";
-private _usingCannon = currentweapon _heli in ["fza_m230", "fza_burstlimiter", "fza_gun_safe"];
+private _usingCannon = currentweapon _heli in ["fza_m230", "fza_burstlimiter"];
```

Comment:
If we are no longer having different safe modes, we need to change how we decide whether we are `_usingCannon` or not.

This introduces a bug where I can WAS gun, be in safe mode, set sight to FXD and not have the "GUN FIXED" warning appear

Reply:
If the removal of the safe modes were not absolutely required for ace hellfire, I would recomment to not remove the safe modes in this PR, then we can worry about this one in the future